CHANGELOG.md

```diff
@@ -276,6 +276,27 @@ Additions to existing modules
   filter-↭ : ∀ (P? : Pred.Decidable P) → xs ↭ ys → filter P? xs ↭ filter P? ys
   ```
 
+* In `Data.List.NonEmpty.Properties`:
+  ```agda
+  ∷→∷⁺ : ∀ {x y : A} {xs ys : List A} →
```

Comment:
We don't tend to put the implicits in the CHANGELOG, (We don't tend to use qualified names either but, in this case, it's probably warranted).